ms/MeasurementSets/MSIter.cc

```diff
@@ -361,6 +361,7 @@ void MSIter::origin()
 {
   curMS_p=0;
   checkFeed_p=True;
+  if (timeComp_p) timeComp_p->setOffset(0.0);

```

Comment:
The call to setState already resets timeComp_p's offset, so this addition appears to be redundant.